### ISSUE TITLE
[BUG] <title>

### ISSUE THREAD

**Issue by gidisegal** [nathom/streamrip#278]
### Describe the bug

after fresh install of streamrip this happens

  Exception

  Device authorization failed {'status': 401, 'error': 'invalid_client', 'sub_status': 1005, 'error_description': 'Client with token Pzd0ExNVHkyZLiYN not found'}

  at /opt/homebrew/lib/python3.9/site-packages/streamrip/clients.py:882 in _get_device_code
       878│         }
       879│         resp = self._api_post(f"{TIDAL_AUTH_URL}/device_authorization", data)
       880│ 
       881│         if resp.get("status", 200) != 200:
    →  882│             raise Exception(f"Device authorization failed {resp}")
       883│ 
       884│         self.device_code = resp["deviceCode"]
       885│         self.user_code = resp["userCode"]
       886│         self.user_code_expiry = resp["expiresIn"]

If this was unexpected, please open a Bug Report at https://github.com/nathom/streamrip/issues/new/choose

### Command Used

```bash
rip url https://tidal.com/browse/track/128385944
```

### Debug Traceback

```text
Exception

  Device authorization failed {'status': 401, 'error': 'invalid_client', 'sub_status': 1005, 'error_description': 'Client with token Pzd0ExNVHkyZLiYN not found'}

  at /opt/homebrew/lib/python3.9/site-packages/streamrip/clients.py:882 in _get_device_code
       878│         }
       879│         resp = self._api_post(f"{TIDAL_AUTH_URL}/device_authorization", data)
       880│ 
       881│         if resp.get("status", 200) != 200:
    →  882│             raise Exception(f"Device authorization failed {resp}")
       883│ 
       884│         self.device_code = resp["deviceCode"]
       885│         self.user_code = resp["userCode"]
       886│         self.user_code_expiry = resp["expiresIn"]

If this was unexpected, please open a Bug Report at https://github.com/nathom/streamrip/issues/new/choose
```

### Config File

```toml
fresh install of streamrip
```

### Operating System

macos

### streamrip version

1.9.2

**Comment by CanadianBaconBoi** [comment]
Solved by #279

**Comment by nathom** [comment]
Fixed in `v1.9.3`.
